File: src/main/java/site/gachontable/domain/waiting/port/in/GetStatus.java
package site.gachontable.domain.waiting.port.in;

import site.gachontable.infra.security.principal.AuthDetails;
import site.gachontable.presentation.waiting.dto.response.StatusResponse;

import java.util.List;

public interface GetStatus {

    List<StatusResponse> execute(AuthDetails authDetails);
}

File: src/main/kotlin/site/gachontable/domain/waiting/port/in/GetStatus.kt
package site.gachontable.domain.waiting.port.`in`

import site.gachontable.infra.security.principal.AuthDetails
import site.gachontable.presentation.waiting.dto.response.StatusResponse

interface GetStatus {
    fun execute(authDetails: AuthDetails): MutableList<StatusResponse>
}
